website_anchor_smooth_scroll/__openerp__.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Antiun Ingeniería S.L. - Jairo Llopis
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Smooth Scroll for Website Anchors',
+    'summary': '''Replace default behavior for internal links (anchor only)
+     with smooth scroll
+    ''',
+    'version': '9.0.1.0.0',
+    'category': 'Website',
+    'website': 'https://laslabs.com/',
```

Comment:
We should leave the original website as attribution to the original authors. We merely upgraded this module, which is nothing in comparison to creating IMO